index.html

```diff
@@ -35,7 +35,7 @@
 		 'SOMEONE SENDS ENV SUPPORT STEP-BY-STEP SOLUTION AND AFTER 30 MINUTES THEY KNOW HOW TO FIX IT', 
 		 'GLOBAL BROKER INCIDENT', 'NOT FOLLOWING RELEASE NOTES', 'PROPERTY SET TO REALLY STRANGE VALUE', 
 		 'RETRO/SPRINT PLANNING POSTPONED', 'AUTOMATION FRAMEWORK FAILS', 'JEDIS DOWN', 'DEPLOYMENT FAILED', 'WEBLAUNCHER ISSUE',
-		 'NGINX ISSUE', 'CONTENT TEAM ARGUES'];
+		 'NGINX ISSUE', 'CONTENT TEAM ARGUES', '502 GATEWAY'];

```

Comment:
It's already included in 'NGINX ISSUE'

Reply:
NGINX ISSUE does not stand for 502 only. It could cover any 4xx or 5xx http codes. And yet we have 403 stated here